docs/markdown/connecting-to-testnet.md

```diff
@@ -0,0 +1,58 @@
+# Connecting to an ILP Testnet
+
+This tutorial will help you start an ILP node locally on your computer and connect to an ILP testnet node.
```

Comment:
What about instead being specific about the components, like this:

"This tutorial will help you start an moneyd locally on your computer and connect to an ILP testnet Connector."

Reply:
https://github.com/interledger/interledger.github.io/pull/108/commits/7fe7c009db48ec2dc6b03b772041d7173b9d010d#diff-95b449acc2e692458b91372a5b6e33e7R3